app/Http/Resources/RequestWithdrawalDetailedResource.php

```
@@ -0,0 +1,34 @@
+            'warehouse_name' => $this->warehouse?->name,
+            'charging_name'  => $this->chargeable_name ?? null,
+            'requested_by'   => $this->created_by_user_name ?? null,
+            'equipment_no'   => $this->equipment_no,
+            'smr'            => $this->smr,
+            'fuel'           => $this->fuel,
+            'items'          => $this->whenLoaded(
+                'items',
+                fn () => RequestWithdrawalItemDetailedResource::collection($this->items),
+                []
+            ),
+            'approvals' => ApprovalAttributeResource::collection($this->approvals)
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\ApprovalAttributeResource' in method 'toArray'. (undefined)

(StaticAccess)